tools/go-cli/go-whisk-cli/commands/wsk.go

```diff
@@ -50,8 +50,8 @@ func init() {
 
     WskCmd.PersistentFlags().BoolVarP(&flags.global.verbose, "verbose", "v", false, "verbose output")
     WskCmd.PersistentFlags().BoolVarP(&flags.global.debug, "debug", "d", false, "debug level output")
-    WskCmd.PersistentFlags().StringVarP(&flags.global.auth, "auth", "u", "", "authorization key")
-    WskCmd.PersistentFlags().StringVar(&flags.global.apihost, "apihost", "", "whisk API host")
-    WskCmd.PersistentFlags().StringVar(&flags.global.apiversion, "apiversion", "", "whisk API version")
+    WskCmd.PersistentFlags().StringVarP(&flags.global.auth, "auth", "u", "", "authorization `key`")
```

Comment:
For the action create commands the words "key, host, and version" are capitalized. We should be consistent with the capitalization of keywords across commands.